src/feedback-screen.tsx

```diff
@@ -77,7 +78,7 @@ const useStyles = createUseStyles((theme) => ({
 
     innerContainer: {
         textAlign: 'left',
-        padding: '64px 24px 16px',
+        padding: '64px 8px 16px',
```

Comment:
shouldn't this produce changes in mobile screenshots?

Reply:
nop, I've added a `<ResponsiveLayout />` wich adds `16px` horizontal padding in mobile: `16 + 8 = 24`